english/rib-eye/rib-eye.md

```diff
@@ -0,0 +1,13 @@
+All this talk of chocolate and donuts. You want a steak. A thick, juicy, salty steak. But, what kind? How about a Rib Eye? But where dost thou find a slab of beautiful bovine around here? 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.